src/Tasks/ContentReviewEmails.php

```diff
@@ -165,9 +167,8 @@ protected function notifyOwner($ownerID, SS_List $pages)
      */
     protected function getEmailBody($config, $variables)
     {
-        $template = SSViewer::fromString($config->ReviewBody);
-        $value = $template->process(ArrayData::create($variables));
-
+        $engine = Injector::inst()->create(TemplateEngine::class);
+        $value = $engine->renderString($config->ReviewBody, ViewLayerData::create(ArrayData::create($variables)));
```

Comment:
Use `TemplateEngine` instead of explicitly `SSTemplateEngine` because `ReviewBody` could be updated to contain whatever syntax the devs wanna use.